README.md

```diff
@@ -10,3 +10,5 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+Assume that two graphs A and B are isomorphic. Further assume that they must be completey connected. Since A & B are both isomorphic, there exists a bijective function that ensures an adjacency relationship between the nodes of both graphs. Consider the case in which A is a complete graph with n nodes (K_{n}) and B is also a complete graph with p nodes, (K_{p}) where n \neq p. Under the asssumption, A and B are isomorphic. Therefore, there must exist a one-to-one correspondence between their nodes. However, we have n nodes in A and p nodes in B where n \neq p. If we have n nodes in A and m nodes in B, and the two are not equal, then it is impossible to find a binective function mapping the nodes of A to the nodes of B while preserving the adjacency relationship. This is because in a complete graph, each node must be connected to every other node: in a situation in which one graph has more nodes than the other, there is bound to be a node(s) in one graph with more connections than the corresponding node(s) in the other. This contradiction arises from the assumption that A and B must be completely connected if they are isomorphic. therefore, graphs A and B need not be completely connected to be isomorphic.
```

Comment:
This sounds like you meant to write this for the other proof? Not a proof for this one though -- you can't assume that they are isomorphic and have a different number of nodes at the same time. Also unclear how completely connected is related to this.